tools/RAiDER/delayFcns.py

```diff
@@ -286,8 +287,8 @@ def getProjFromWMFile(wm_file):
     '''
     Returns the projection of an HDF5 file
     '''
-    with h5py.File(wm_file, 'r') as f:
-        wm_proj = CRS.from_json(f['Projection'][()])
+    with Dataset(wm_file, mode='r') as f:
```

Comment:
Same thing here. We should have a NETCDF "projection reader" that compliments rather than replaces the HDF5 reader.